.github/actions/templates/validateModulePester/action.yml

```diff
@@ -172,6 +164,6 @@ runs:
 
     - name: 'Publish Test Results'
       uses: EnricoMi/publish-unit-test-result-action@v1
-      if: always() && !contains('cancelled,skipped', steps.pester_api_run_step.outcome)
+      if: always()
```

Comment:
Just checking, any alignment needed for ADO or not applicable?

Reply:
No. The ADO publish task is the default native - that works like a charm.